docusaurus/docs/plugins/plugins-getting-started.md

```diff
@@ -104,148 +96,90 @@ yarn dev
-
-## Use Case: Dynamically loading a UI Plugin
+Note, that you should be able to make changes to the extension and the UI will hot-reload and update in the browser.
 
-In the previous use case, the UI package we created was statically built into the UI - this works great for the developer use case where we want to be able to iterate, make changes and see those via hot-reload as we do with Rancher Dashboard today.
+## Building the Extension
 
-This use case illustrates being able to build a UI plugin as a package and then be able to load that into the UI, dynamically at run-time.
+Up until now, we've run the extension inside of the skeleton application - this is the developer workload.
 
-First thing to do is to build the UI package for the `testplugin` plugin that we created previously. Run:
+To build the extension so we can use it independently, run:
 
 ```
-yarn build-pkg testplugin
+yarn build-pkg test
 ```
 
-The `testplugin` plugin will be built and the output placed in `dist-pkg\testplugin`.
+This will build the extension as a Vue library and the built extension will be placed in the `dist-pkg` folder.
 
-Next, edit the `nuxt.config.js` file in the root folder and replace it with this:
+## Loading the Extension
 
-```
-import config from './shell/nuxt.config';
-
-// Excludes the following plugins if there's no .env file.
-const excludes = process.env.EXCLUDES_PKG || 'epinio, rancher-components, testplugin';
+When we run `yarn dev` at the moment, our test extension will be automatically loaded in the application - this allows us to develop
+the extension with hot-reload.
 
-export default config(__dirname, {
-  excludes: excludes.replace(/\s/g, '').split(','),
-  // excludes: ['fleet', 'example']
-  // autoLoad: ['fleet', 'example']
-});
-```
+To test loading the extension dynamically, we can update configuration to tell Rancher not to include our extension.
 
-**NOTE**: If you're actively working with Plugins, we can make use of the `.env` file instead of hardcoded the excludes in the `nuxt.config.js` file so your changes on the file won't be tracked by Git.
+To do this, create a new `.env` file in the root `test-app` folder, and add these contents:
 
 ```
-# First, create the .env file from template
-cp env.example .env
-
-# Then, add the following to the .env file
-EXCLUDES_PKG='epinio, rancher-components, testplugin'
+EXCLUDES_PKG='test'
 ```
 
-What we have done here is add the `testplugin` plugin to the `excludes` configuration - this means that when we build and run the UI, we won't include the `testplugin` plugin.
+If necessary, bring in the environment variables by running `source .env`.
 
-Run the UI with:
+Now, run the UI with:
 
 ```
 yarn dev
 ```
 
-Open a web browser to `https://127.0.0.1:8005` and you'll see that the Example nav item is not present - since the plugin was not loaded.
+Open a web browser to `https://127.0.0.1:8005` and you'll see that the Example nav item is not present - since the extension was not loaded.
```

Comment:
This is a link here, above it's an inline `code block`

Reply:
Made them all links